content/references/rippled-api/public-rippled-methods/nft-methods/nft_buy_offers.md

```diff
@@ -0,0 +1,111 @@
+---
+html: nft_buy_offers.html
+parent: nft-methods.html
+blurb: Get a list of all buy offers for a NFToken.
+labels:
+  - Non-fungible Tokens, NFTs, NFTokens
+---
+# account_nfts
+[[Source]](https://github.com/ripple/rippled/blob/xls20/src/ripple/rpc/handlers/NFTOffers.cpp "Source")
+{% include '_snippets/nfts-disclaimer.md' %}
+
+The `nft_buy_offers` method returns a list of buy offers for a `NFToken` object.
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+   "id": 1,
+   "command": "nft_buy_offers",
+   "params": [{
+   		"tokenid": "00090000D0B007439B080E9B05BF62403911301A7B1F0CFAA048C0A200000007"
+   }]
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "nft_buy_offers",
+    "tokenid": "00090000D0B007439B080E9B05BF62403911301A7B1F0CFAA048C0A200000007"
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+[Try it! >](websocket-api-tool.html#nft_sell_offers)
+
+The request includes the following parameters:
+
+| Field                 | Type                       | Description             |
+|:----------------------|:---------------------------|:------------------------|
+| `tokenid`             | String                     | The unique identifier of a NFToken object. |
+|
```

Comment:
There are other possible parameters for the request method; from my quick testing, `ledger_index` and `ledger_hash` are both possible. The [source code comments](https://github.com/ripple/rippled/blob/161bd6e9ad3fe25c0c6c5718835586124b9c1e2c/src/ripple/rpc/handlers/NFTOffers.cpp#L43-L44) suggest that `marker` and `limit` are also allowed, per the standard "markers and pagination" rules.